### ISSUE TITLE
Release 1.2.0

### ISSUE THREAD

**Issue by ramijarrar** [Zodiase/meteor-mdl#47]
https://github.com/google/material-design-lite/releases/tag/v1.2.0

**Comment by Zodiase** [comment]
@ramijarrar Thanks for the notice. I'm just a bit busy this week. Should be able to get the new release out before this weekend. :)

**Comment by Zodiase** [comment]
So umm... I've encountered something strange.

But firstly, the update is out on `zodiase:mdl-edge@1.2.0` for testing.

Then there's the problem. I don't see any issue with my installing the edge version. But on Travis CI, [as you can see here](https://travis-ci.org/Zodiase/meteor-mdl/builds), the last few builds have been failing for this same error:

``` shell
=> Build failed:

   While checking for fourseven:scss@3.8.1:
   error: No compatible binary build found for this package. Contact the
   package author and ask them to publish it for your platform.
```

And there's [already an issue open for this](https://github.com/fourseven/meteor-scss/issues/201).

So at the moment I'm not sure if I should release update 1.2.0 on the main package.

What do you think?

**Comment by zhanbst** [comment]
Can you check fourseven:scss@3.9.0 is available for linux version meteor 1.4.1.1

**Comment by Zodiase** [comment]
@zhanbst When I tried `meteor add fourseven:scss` I got version `3.9.0` so I'd say no. Meanwhile its github readme says:

> Since meteor 1.4.1+ (fourseven:scss 3.9.0+), we do not have prebuild binaries anymore. You are required to set up the required toolchain yourselves.

which I'm not quite sure what to do about.

**Comment by Zodiase** [comment]
@zhanbst Update: When I tried `meteor update fourseven:scss` I successfully got version `3.9.0` even though for some reason it wasn't the default version when adding. Sorry for the confusion but `3.9.0` is available for linux version meteor `1.4.1.1`. Tested on my ubuntu VM.

**Comment by Zodiase** [comment]
`1.2.0` is out. 617050499ccf61da234c6467244df55b34438814
